src/components/header/_.module.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 7px 15px;
+}
+
+.icon {
+  width: 17px;
+  height: 17px;
+}
+
+.title {
+  font-size: 18px;
+  font-weight: 400;
+  margin-bottom: 3px;
```

Comment:
why do we need margin bottom here?
![image](https://user-images.githubusercontent.com/82516098/123785108-d273a400-d8e0-11eb-86fc-22658635b71d.png)
looks like we have different margin, so I think we can delete it

Reply:
It's for vertical-align of title relatively to image and icon